app/queries/base_query.rb

```diff
@@ -22,6 +22,19 @@ def initialize(organization:, pagination: DEFAULT_PAGINATION_PARAMS, filters: {}
 
   attr_reader :organization, :pagination_params, :filters, :search_term, :order
 
+  def validate_filters
+    contract_class_name = "#{self.class.name}FiltersContract"
+    contract = Object.const_get(contract_class_name).new
```

Comment:
Just a thought, do you think we could find a way to pass or assign the Contract class to the query class, to avoid doing some magic to load it?